docs/user_guide/application.md

```diff
@@ -21,32 +20,32 @@ class MyApplication(JupyterApp):
 Register an event schema with the logger.
 
 ```python
-        schema = """
-        $id: http://myapplication.org/my-method
-        version: 1
-        title: My Method Executed
-        description: My method was executed one time.
-        properties:
-        msg:
-            title: Message
-            type: string
-        """
-
-        self.eventlogger.register_event_schema(
-            schema=schema
-        )
```

Comment:
The indent here was intentional, since it's part of the method from the preview code chunk.

Reply:
Fixed